docs/pages/access-controls/access-request-plugins/access-monitoring-rules.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: Routing Access Request notifications
+description: How to set up Teleport's Access Monitoring Rules to route Access Request notifications
+---
+
+With Teleport's Access Monitoring Rules Access Request notifications
+can be routed to plugins based on based on several conditions.
+For example, if you wish to notify different recipients e.g Slack channels,
+based on what roles or resources are being accessed, or based on what user
+is making the request.
+
+Currently only the Slack access plugin is supported.
+
+## Prerequisites
+
+- A managed Teleport Enterprise account.
+
+- The Enterprise `tctl` admin tool and `tsh` client tool version >= (=teleport.version=).
+
+  You can verify the tools you have installed by running the following commands:
+
+  ```code
+  $ tctl version
+  # Teleport Enterprise v(=teleport.version=) go(=teleport.golang=)
+  
+  $ tsh version
+  # Teleport v(=teleport.version=) go(=teleport.golang=)
+  ```
+
+  You can download these tools by following the appropriate [Installation 
+  instructions](../../installation.mdx#linux) for your environment and Teleport edition.
+
+- At least one of the Teleport Access Plugins that support using being set up.
+
+- (!docs/pages/includes/tctl.mdx!)
+
+## Step 1/2. Create an Access Monitoring Rule
+
+You can define an Access Request notification rule in two ways:
+- Using the web UI dialog.
+- Creating a YAML resource file containing the definition of a rule.
+
+### Create an Access Monioting Rule via the web UI
+
+To create an Access Monitoring Rule via the web UI, first navigate to
+the Access Request page and click on `View Notification Routing Rules`.
+
+![View Notification Routing Rules](../../../img/enterprise/plugins/access_monitoring_rules/view_routing_rules.png)
+
+Then click `Create Notification Rule`.
+
+![Create AMR](../../../img/enterprise/plugins/access_monitoring_rules/access_monitoring_rule_ui.png)
+
+If a plugin that supports Access Monitoring Rule based routing is not enrolled
+the UI will prompt you to enroll one.
+
+### Create an Access Monitoring Rule using YAML resource
+
+Here is an example Access Monitoring Rule. This rule will notify `your_slack_channel`
+via the Slack plugin if an Access Request containing the role `your_role_name` is
+made.
+
+```yaml
+kind: access_monitoring_rule
+version: v1
+metadata:
+  name: your-plugin-name
+spec:
+  subjects: ['access_request']
+  condition: 'access_request.spec.roles.contains("your_role_name")'
+  notification:
+    name: 'slack'
+    recipients: ['your_slack_channel']
+```
+
+Above is an example of an Access Monitoring Rule that routes incoming Access Requests
+that are requesting the role `your_role_name` to the slack channel `your_slack_channel`.
```

Comment:
You've already said exactly his in the previous paragraph.

Reply:
I'll remove it here and leave the one above